src/autocomplete.js

```diff
@@ -271,6 +275,10 @@ export default class Autocomplete extends Component {
       query: newQuery,
       selected: -1
     })
+
+    if (this.props.onChange) {
```

Comment:
You can avoid this check by defining `onChange` in `defaultProps` as `() => {}`.

This is consistent with `onConfirm` so I think we should do this.

Also will need a README update.